.github/workflows/release-docker.yml

```diff
@@ -40,4 +40,8 @@ jobs:
 
       - run: |
           version="$(cat ./pkg/urbit/version)"
-          $(nix-build -A skopeo)/bin/skopeo --insecure-policy copy tarball:$(nix-build -A docker-image) docker://${{ secrets.DOCKERHUB_USERNAME }}/urbit:v$version
+          skopeo="$(nix-build -A skopeo)/bin/skopeo"
+          image="$(nix-build -A docker-image)"
+          $skopeo --insecure-policy copy tarball:$docker docker://${{ secrets.DOCKERHUB_USERNAME }}/urbit:v$version 
```

Comment:
Checked it this morning and realized there is a typo: should be `tarball:$image` instead of `tarball:$docker`

I wish there was a better way to test GitHub CI actions.

Reply:
I will make one more PR to fix this @jtobin

Reply:
Ah, my bad for not catching it.  I went over the changes carefully, but not quite carefully enough. 😄